awx/settings/defaults.py

```diff
@@ -420,10 +420,18 @@ def IS_TESTING(argv=None):
 # Set default ports for live server tests.
 os.environ.setdefault('DJANGO_LIVE_TEST_SERVER_ADDRESS', 'localhost:9013-9199')
 
+# heartbeat period can factor into some forms of logic, so it is maintained as a setting here
+CLUSTER_NODE_HEARTBEAT_PERIOD = 60
+RECEPTOR_SERVICE_ADVERTISEMENT_PERIOD = 60  # https://github.com/ansible/receptor/blob/aa1d589e154d8a0cb99a220aff8f98faf2273be6/pkg/netceptor/netceptor.go#L34
```

Comment:
from discussing with @fosterseth 

If you follow the use of this setting, you can paint a picture of worst-case timing. The execution node is seen by the receptor mesh at 1 second after 1pm, so 13:00:01, which just missed the heartbeat. The control node runs its next heartbeat at 13:01:00.

But there might be a timing issue where control node 1 and control node 2 run almost at the same time. Anyway, we add these numbers together because we want to be _sure_ that the node is lost before marking its capacity to 0.